packages/zosfiles/__tests__/__integration__/cli/create/zfs/__snapshots__/cli.files.create.zfs.integration.test.ts.snap

```diff
@@ -0,0 +1,161 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Create z/OS File System should display the help 1`] = `
+"================Z/OS FILES CREATE ZOS-FILE-SYSTEM HELP===============
+
+ COMMAND NAME
+ ------------
+
+   zos-file-system | zfs
+
+ DESCRIPTION
+ -----------
+
+   Create a z/OS file system
+
+ USAGE
+ -----
+
+   zowe zos-files create zos-file-system <fileSystemName> [options]
+
+ POSITIONAL ARGUMENTS
+ --------------------
+
+   fileSystemName		 (string)
+
+      The name of the file system to mount
```

Comment:
This looks incorrect for the description of the create command. You might be sharing options between commands where the context is different (create vs. mount).